src/index.ts

```diff
@@ -32,12 +32,12 @@ export class GraphQLClient {
     const result = await getResult(response)
 
     if (response.ok && !result.errors && result.data) {
-      return result
+      return { ...result, headers: response.headers }
```

Comment:
Couldn't there be a clash if `result` contains a field called `headers`?

Reply:
Yeah this is a good point. Should we use a prefix like underscore with a special name or are there any other suggestions? For example: `_headers`, `_responseHeaders`?